app/client/lib/koForm.css

```diff
@@ -203,6 +214,17 @@ div:hover > .kf_tooltip {
   color: #777777;
 }
 
+.drag_delete.icon-remove::before {
+  display: block;
+  background-color: var(--grist-theme-control-secondary-fg, var(--grist-color-slate));
+  content: ' ';
+  mask-image: var(--icon-Remove);
+  width: 14px;
+  height: 14px;
+  mask-size: contain;
+  mask-repeat: no-repeat;
+}
+
```

Comment:
Since this mostly shares the same styles as above, maybe we can break out the common stuff into a new `.kf_draggable__icon` rule?